qupath-core-processing/src/main/java/qupath/lib/scripting/QP.java

```diff
@@ -1410,9 +1410,6 @@ public static void setChannels(ImageChannel... channels) {
 	 */
```

Comment:
Could you update the Javadoc? (not possible to set channels **colors** for RGB images)